### ISSUE TITLE
Update httpx

### ISSUE THREAD

**Issue by ognone** [huge-success/sanic#1786]
[httpx](https://github.com/encode/httpx) 0.11.1 (sanic demands 0.9.3) fails with:

```
  File "/usr/lib/python3.8/site-packages/sanic/__init__.py", line 2, in <module>
    from sanic.app import Sanic
  File "/usr/lib/python3.8/site-packages/sanic/app.py", line 35, in <module>
    from sanic.testing import SanicASGITestClient, SanicTestClient
  File "/usr/lib/python3.8/site-packages/sanic/testing.py", line 188, in <module>
    class SanicASGIDispatch(httpx.dispatch.ASGIDispatch):
AttributeError: module 'httpx.dispatch' has no attribute 'ASGIDispatch'
```

A simple fix to `class SanicASGIDispatch(httpx.ASGIDispatch):` in line 188 of testing.py fixes this

**Comment by syfluqs** [comment]
Also, need to bump up the version of httpx to 0.11.1 for sanic. This dependency was added later in sanic 19.12LTS and breaks httpx AsyncClient dependent code. AsyncClient is not present in httpx 0.9.3 and was added after this version.

**Comment by ashleysommer** [comment]
Fixed by https://github.com/huge-success/sanic/pull/1794

**Comment by mikeyusko** [comment]
@Tronic Should we close the issue? seems like #1794 fixed it and the PR had have merged.

**Comment by Tronic** [comment]
This one may be closed but httpx>=0.12 won't work until #1806 is merged as well.
